Sources/Datadog/Utils/SwiftExtensions.swift

```diff
@@ -101,3 +101,41 @@ extension Array {
         0 <= index && index < count ? self[index] : nil
     }
 }
+
+// MARK: - Time Convinience
+
+public extension TimeInterval {
+    var seconds: TimeInterval { return TimeInterval(self) }
+    var minutes: TimeInterval { return self.multiplyOrClamp(by: 60) }
+    var hours: TimeInterval { return self.multiplyOrClamp(by: 60.minutes) }
+    var days: TimeInterval { return self.multiplyOrClamp(by: 24.hours) }
```

Comment:
Similar, this should be part of `Sources/Datadog/DatadogInternal/**` so we don't pollute user namespaces with our utility code (now, by importing `Datadog` they will see all these helpers in V2).